dashboard/.eslintrc.json

```diff
@@ -0,0 +1,89 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 2020,
+    "sourceType": "module",
+    "project": "./tsconfig.json"
+  },
+  "settings": {
+    "react": {
+      "version": "detect"
+    },
+    "import/resolver": {
+      "typescript": {}
+    }
+  },
+  "extends": [
+    "react-app",
+    "react-app/jest",
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:jsx-a11y/recommended",
+    "plugin:jest/recommended",
+    "plugin:import/errors",
+    "plugin:import/warnings",
+    "prettier",
+    "plugin:react/recommended",
+    "plugin:prettier/recommended"
+  ],
+  "plugins": ["react", "@typescript-eslint", "jest", "jsx-a11y", "import"],
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "rules": {
+    "quotes": [
+      "error",
+      "double",
+      {
+        "allowTemplateLiterals": true,
+        "avoidEscape": true
+      }
+    ],
+    "import/no-anonymous-default-export": [
+      "error",
+      {
+        "allowObject": true,
+        "allowArrowFunction": true
+      }
+    ],
+    "@typescript-eslint/ban-types": [
+      "error",
+      {
+        "extendDefaults": true,
+        "types": {
+          "{}": false, // allow {} as a type as it is safe to be used in React
+          "object": false // allow object as a type
+        }
+      }
+    ],
+    "@typescript-eslint/explicit-module-boundary-types": "off", // disable warning when missing return type
+    "@typescript-eslint/no-empty-function": [
+      "error",
+      {
+        "allow": [
+          "arrowFunctions" // allow () => {}
+        ]
+      }
+    ],
+    "@typescript-eslint/no-explicit-any": "off", // allow using 'any'
+    "@typescript-eslint/no-non-null-assertion": "off", // allow non-null assertions using '!'
+    "jest/no-conditional-expect": "off", // allow 'expect' in 'if' and 'try-catch'
+    "jest/valid-title": "off", // allow dynamic test names (e.g., 't.name')
+    "jsx-a11y/label-has-associated-control": "off", // TODO(agamez): check if <label><customelement></label> works
+    "jsx-a11y/no-onchange": "off", // TODO(agamez): allow using onChange instead of onBlur, but it is not recommended
```

Comment:
the `no-onchange` rule seems deprecated so no need to activate it: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-onchange.md

Regarding `label-has-associated-control` I know this is enforced by Clarity Core components. Have you tried if it fails if you enable it? (It shouldn't be a problem but maybe we are missing some case, I am okay doing that in other PR).

Reply:
I've just tried to disable the rule. Most of them are false positives since Clarity takes care of the a11y stuff under the hood. Consequently, I've disabled the rule in the corresponding files, but it is enabled to still detect these problems when not using cds components.